src/content/en/updates/2017/03/chrome-58-media-updates.md

```diff
@@ -0,0 +1,198 @@
+[Chromestatus Tracker](https://www.chromestatus.com/features/5651339115757568) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=690468)
+
+## color-gamut Media Query {: #colorgamut }
+
+As wide color gamut screens are more and more popular, sites can now access the
+approximate range of colors supported by Chrome and output devices using the
+`color-gamut` Media Query.
+
+If you're not familiar yet with the definition of Color Space, Color Profile,
+Gamut, Wide-gamut and Color Depth, I highly recommend you read the 
+[Improving Color on the Web] WebKit blog post. It goes into much detail on how
+to use of the `color-gamut` Media Query to serve a wide-gamut image when user
+is on a wide-gamut display, and fallbacks to an sRGB image.
+ 
+The current implementation in Chrome accepts the `srgb`, `p3` (gamut specified
+by the DCI P3 Color Space), and `rec2020` (gamut specified by the ITU-R
+Recommendation BT.2020 Color Space) keywords. Check out the [official
+sample](https://googlechrome.github.io/samples/media/color-gamut-media-query.html).
+
+Usage in HTML
+
+<pre class="prettyprint lang-html">
+&lt;picture>
+  &lt;source media="(color-gamut: p3)" srcset="photo-p3.jpg">
+  &lt;source media="(color-gamut: rec2020)" srcset="photo-rec2020.jpg">
+  &lt;img src="photo-srgb.jpg">
+&lt;/picture>
+</pre>
+
+Usage in CSS
+
+    main {
+      background-image: url("photo-srgb.jpg");
+    }
+    
+    @media (color-gamut: p3) {
+      main {
+        background-image: url("photo-p3.jpg");
+      }
+    }
+    
+    @media (color-gamut: rec2020) {
+      main {
+        background-image: url("photo-rec2020.jpg");
+      }
+    }
+
+Usage in JavaScript
+
+    // It is expected that the majority of color displays will return true.
+    if (window.matchMedia("(color-gamut: srgb)").matches) {
+      document.querySelector('main').style.backgroundImage = 'url("photo-srgb.jpg")';
+    }
+    
+    if (window.matchMedia("(color-gamut: p3)").matches) {
+      document.querySelector('main').style.backgroundImage = 'url("photo-p3.jpg")';
+    }
+
+    if (window.matchMedia("(color-gamut: rec2020)").matches) {
+      document.querySelector('main').style.backgroundImage = 'url("photo-rec2020.jpg")';
+    }
+
+{% framebox height="240px" %}
+<p>For info, this screen currently supports approximately:</p>
+<ul>
+  <li id="srgb" hidden>
+    the sRGB gamut or more.
+  </li>
+  <li id="p3" hidden>
+    the gamut specified by the DCI P3 Color Space or more.
+  </li>
+  <li id="rec2020" hidden>
+    the gamut specified by the ITU-R Recommendation BT.2020 Color Space or more.
+  </li>
+</ul>
+<script>
+  document.querySelector('#srgb').hidden = !window.matchMedia("(color-gamut: srgb)").matches;
+  document.querySelector('#p3').hidden = !window.matchMedia("(color-gamut: p3)").matches;
+  document.querySelector('#rec2020').hidden = !window.matchMedia("(color-gamut: rec2020)").matches;
+</script>
+{% endframebox %}
```

Comment:
Fixed in https://github.com/google/WebFundamentals/pull/4335/commits/cef4bdcf3738faa4cc6ce9518d10aa2f904a6db6